simapp/go.mod

```diff
@@ -210,4 +210,5 @@ replace (
 	// Fix upstream GHSA-h395-qcrw-5vmq vulnerability.
 	// TODO Remove it: https://github.com/cosmos/cosmos-sdk/issues/10409
 	github.com/gin-gonic/gin => github.com/gin-gonic/gin v1.8.1
+	github.com/syndtr/goleveldb => github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7
```

Comment:
Can we add a comment on why it is required here too.

Reply:
On it, I also forgot to do the replace in the main go.mod. Not sure if that's gonna make any difference, but just to be safe I'll replace it